home_work_2/seq_master.py

```diff
@@ -0,0 +1,106 @@
+def check_sequence(new_seq: str):
+    """ The function takes a string with nucleotide sequence, checks if there is at least one unexpected character,
+    or the sequence is ambiguous (U and T are both there),
+    returns the answer if the sequence is correct (true or false) and problem_source if any (Mistake, Ambiguous) or ""
+    if there is no problem.
+
+    :param new_seq: string
+    :return: correct (True/False), problem_source("Mistake", "Ambiguous", "")
+    """
+    correct, problem_source = True, ""
+
+    for nucl in new_seq:
+        if nucl not in ["A", "T", "G", "C", "a", "t", "g", "c", "U", "u", "N", "n"]:
+            correct, problem_source = False, "Mistake"
+            break
+    if correct:
+        if ("U" in new_seq or "u" in new_seq) and ("T" in new_seq or "t" in new_seq):
```

Comment:
Снова проще на множествах
```suggestion
        if {'u', 't'} <= set(new_seq.lower()):
```